### ISSUE TITLE
Clarify Relation Name uniqueness

### ISSUE THREAD

**Issue by Verschwiegener** [mvrdevelopment/spec#235]
**Is your feature request related to a problem? Please describe.**
The Relation Attribute Table specifies that the Relation Name should be unique (https://github.com/mvrdevelopment/spec/blob/main/gdtf-spec.md#table-63-relation-attributes) but in the example below the Relations all have the same Name. Should the Name be unique?

**Comment by petrvanekrobe** [comment]
Yes, it should be unique, the Builder enforces this. Thank you for opening the issue.
